lib/media/preload_manager.js

```diff
@@ -368,6 +368,32 @@ shaka.media.PreloadManager = class extends shaka.util.FakeEventTarget {
     return this.chooseInitialVariantPromise_;
   }
 
+  /**
+   * @param {?shaka.extern.AbrManager} abrManager
+   */
+  attachAbrManager(abrManager) {
+    this.abrManagerEntrusted_ = true;
```

Comment:
This should not set `this.abrManagerEntrusted_`.
That variable means that the lifecycle of the abr manager is in the hands of a player instance. Ideally, when `unloadAndSavePreload` is called, all of the managers passed to it are detangled from the player that made them.

Reply:
Done!